ui/app/AppLayouts/Profile/popups/ConfirmChangePasswordModal.qml

```diff
@@ -48,11 +48,59 @@ StatusDialog {
 
     width: 480
     height: 546
-    closePolicy: d.passwordChanged ||  d.dbEncryptionInProgress ? Popup.NoAutoClose : Popup.CloseOnEscape | Popup.CloseOnPressOutside
+    closePolicy: d.passwordChanged || d.dbEncryptionInProgress
+                     ? Popup.NoAutoClose
+                     : Popup.CloseOnEscape | Popup.CloseOnPressOutside
+
+    // Overwrite the default modal background with a conditioned blurred one
+    Overlay.modal: Item {
+        Rectangle {
+            anchors.fill: parent
+            visible: !blurredBackground.visible
+            color: Theme.palette.backdropColor
+        }
+        Rectangle {
+            id: blurredBackground
+
+            anchors.fill: parent
+            color: "transparent"
+            visible: d.dbEncryptionInProgress
+
+            // Update the blur source only once, when the modal is shown for performance reasons
+            onVisibleChanged: {
+                if (!visible) {
+                    return;
+                }
+                blurSource.scheduleUpdate()
+            }
+
+            GaussianBlur {
```

Comment:
`FastBlur` would be a better option, exactly for the performance reasons mentioned above :)

Reply:
Thanks for looking into it. In tried `FastBlur` first but it doesn't fit the design (not so uniform blur). The alternative was to render only once the all window which is really negligible.

Reply:
Yeah the FastBlur isn't as nice visually

Reply:
![image](https://github.com/status-im/status-desktop/assets/47554641/42479460-f767-46eb-b72e-a129e5ce7f03)